src/utils/rpc/webSocket.ts

```
@@ -56,6 +56,10 @@ export async function getWebSocketRpcClient(
           close_.bind(socket)()
           onClose()
         },
+        ping() {
+          if (socket.readyState === socket.CLOSED) return
```

Should we throw an error here?

---

done